src/Faker/Provider/Base.php

```diff
@@ -363,7 +363,7 @@ public static function shuffleString($string = '', $encoding = 'UTF-8')
         return implode('', static::shuffleArray($array));
     }
 
-    private static function replaceWildcard($string, $wildcard = '#', $callback = 'static::randomDigit')
+    private static function replaceWildcard($string, $wildcard, $callback)
```

Comment:
Method is `private` and always invoked with arguments for all parameters - no need for default values.

Also see #571.